.github/workflows/cmake_build.yml

```
@@ -109,9 +110,18 @@ jobs:
         JFROG_ARTIFACTORY_NUGET_TOKEN: ${{secrets.JFROG_ARTIFACTORY_NUGET_TOKEN}}
 
     - name: Install dependencies
+      if: runner.os != 'Windows'
       run: |
         exec bash "scripts/ci/dependencies.sh"
 
+    - name: "[Windows] Install dependencies"
+      if: runner.os == 'Windows'
+      shell: pwsh
```

```suggestion
      shell: pwsh
      env:
        SCOOP: C:\Scoop\
```
Please set the environment variable only where it is needed instead of globally for all steps.